server/src/main/java/org/jboss/as/server/controller/resources/DeploymentAttributes.java

```diff
@@ -127,7 +127,7 @@ public void marshallAsAttribute(AttributeDefinition attribute, ModelNode resourc
             .build();
 
     public static final AttributeDefinition STATUS = SimpleAttributeDefinitionBuilder.create(ModelDescriptionConstants.STATUS, ModelType.STRING, true)
-        .setValidator(new EnumValidator<AbstractDeploymentUnitService.DeploymentStatus>(AbstractDeploymentUnitService.DeploymentStatus.class, true))
+        .setValidator(new EnumValidator<AbstractDeploymentUnitService.DeploymentStatus>(AbstractDeploymentUnitService.DeploymentStatus.class))
```

Comment:
For consistency, this should be probably replaced by the EnumValidator.create(..) version. Anyway, it shouldn't block merging this.

Reply:
Hi, I think that originally a new EnumValidator with no values would be created since there are no varargs. Using the EnumValidator.create() method would create an EnumValidator with all enum values, so I decided to use the constructor in order to keep consistent with the original.

Reply:
Hello @clara0 , you're right, thanks for clarifying it. What you have described is correct to me too.

Reply:
Although the deprecation message at the [old constructor](https://github.com/wildfly/wildfly-core/blob/0d4059b79850503ec612e7717faeb119a599be57/controller/src/main/java/org/jboss/as/controller/operations/validation/EnumValidator.java#L94) states that this constructor is equivalent to calling it with `EnumSet#allOf(Class)`

I'm not sure yet how one way or the other could affect it, so, my conclusion of this point is your approach is the safest one.